tests/exporters/onnx/test_export_cli.py

```diff
@@ -730,3 +734,50 @@ def test_complex_synonyms(self):
             model.save_pretrained(tmpdir_in)
 
             main_export(model_name_or_path=tmpdir_in, output=tmpdir_out, task="text-classification")
+
+    @parameterized.expand(_get_models_to_test(PYTORCH_EXPORT_MODELS_TINY, library_name="transformers"))
+    @require_torch_gpu
+    @require_vision
+    @slow
+    @pytest.mark.gpu_test
+    @pytest.mark.run_slow
+    def test_exporters_cli_pytorch_with_slim(
```

Comment:
no need for this to be a gpu test, simply let it be a cpu test, and parametrize it with only a subset of models, something like: ["modernbert", "llama", "t5", "whisper", "stable-diffusion"]. We will add testing for more models when the ONNX export CI is distributed better and faster (as you can see it takes very long right now)

Reply:
Well, onnxslim takes lots of time also, I'm still working to improve it.

Reply:
yes ! did you try doing graph transformations without loading the external data file ? (weights)

Reply:
That's one of the optimization technique that I'm working on, only load paritial weight when it's really needed.

Reply:
tracked here https://github.com/inisis/OnnxSlim/issues/91